app/full_text_search.py

```diff
@@ -149,25 +150,21 @@ def get_person_ids_from_results(query_dict, results_list):
     added_results = set()
     for results in results_list:
         for document in results:
-            field_dict = {field.name: field.value for field in
+            fields = {field.name: field.value for field in
                           document.fields}
-            names = field_dict[
-                'names_romanized_by_romanize_word_by_unidecode']
-            record_id = field_dict['record_id']
-            romanized_jp_names = field_dict['names_romanized_by'
-                                            '_romanize_japanese_word']
-            locations_romanized_unidecode = field_dict['full_location_romanized_by'
-                                   '_romanize_word_by_unidecode']
-            locations_romanized_jp = field_dict['full_location_romanized_by'
-                                             '_romanize_japanese_word']
+
+            record_id = fields['record_id']
+
             # use set to faster the speed.
             # average time complexity: set-O(1) list-O(n)
             if record_id in added_results:
                 continue
 
-            if is_query_match(name_query_txt, names, romanized_jp_names) and \
-                    is_query_match(location_query_txt,
-                                   locations_romanized_unidecode, locations_romanized_jp):
+            romanized_names = [value for name, value in fields.items() if name in romanized_name_fields]
+            romanized_locations = [value for name, value in fields.items() if name in romanized_location_fields]
+
+            if is_query_match(name_query_txt, romanized_names) and \
```

Comment:
Use:
```python
if (... and
        ...):
```
instead of backslash. We prefer using brackets to backslashes to break lines.